### ISSUE TITLE
it take more memory by webp convert to png

### ISSUE THREAD

**Issue by hxming2919** [SDWebImage/SDWebImage#2715]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

/********* main code start  **************/
UIImage *image = [[SDWebImageCodersManager sharedInstance] decodedImageWithData:rep_imageData];

NSData* data = UIImagePNGRepresentation(image);

/********* main  code end  **************/

app get the img with webp url by sdwebimage sdk,
then convert the img  to png  data by UIImagePNGRepresentation.
found that the result png data is bigger than original png data.

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 10 / Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
Designed behavior.

WebP supports lossless compression, but not by default. Which means, if you encode -> decode -> encode multiple times, the size and the quanlity will become worse and worse. 

> found that the result png data is bigger than original png data.

There are no any compare between the two PNG compression data. What's the first one ?

And, since PNG compression, supports the compression quanlity. Which means, even it's lossless, the size of compression data, which using the different quanlity, will be different. Which means:

original bitmap -> [255, 255, 255, 0]

+ quanlity: 100 size: 40KB
+ quanlity: 50 size: 10KB

decoded: same bitmap buffer [255, 255, 255, 0]
